heareval/tasks/util/luigi.py

```diff
@@ -63,6 +81,56 @@ def stage_number(self) -> int:
             raise ValueError("Unknown requires: %s" % self.requires())
 
 
+class DownloadCorpus(WorkTask):
+    """
+    Task for downloading a dataset to a specific filename
+    """
+
+    # URL to download dataset from
+    url = luigi.Parameter()
+    outfile = luigi.Parameter()
+
+    @property
+    def name(self):
+        return type(self).__name__
+
+    def run(self):
+        download_file(self.url, os.path.join(self.workdir, self.outfile))
+        with self.output().open("w") as _:
+            pass
+
+    @property
+    def stage_number(self) -> int:
+        return 0
+
+
+class ExtractZipFile(WorkTask):
```

Comment:
If unpack archive works for many different archive types lets update this name to something more generic like `ExtractArchive` and update variable names below that refer to a zip